### DISCUSSION TITLE
New features for published docs! 🚀

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#771] [Announcements]
Hey everyone,

Today we have five powerful updates to docs sites to share with you! Head to each feature’s discussion page to leave feedback or start a discussion. Here’s what’s new:

**[🗂️ Site sections](https://github.com/orgs/GitbookIO/discussions/767)**: Organize multiple topics on one docs site with tabbed areas — and global search.

**[🎨 Enhanced customization](https://github.com/orgs/GitbookIO/discussions/768)**: Add primary/secondary buttons to your header and break down info with new Stepper blocks.

**[🔒 Hidden pages](https://github.com/orgs/GitbookIO/discussions/769)**: Control access to sensitive content by restricting page visibility.

**[📈 Advanced site insights](https://github.com/orgs/GitbookIO/discussions/770)**: Track performance trends with updated analytics, and stay tuned for AI-driven insights coming soon.

**[💵 New pricing for new users](https://www.gitbook.com/pricing)**: New plans are available for new users. Existing customers can keep their current plan or switch if the new options are a better fit.

Let us know what you think about these new features, and keep an eye out for more announcements tomorrow!

**Comment by JaniceManwiller** [comment]
I wasn't able to find anything in the doc about primary/secondary buttons.

But I am happy to see the view counts back in the insights!

**Reply by addisonschultz** [reply]
You can find the feature under the customization options! 

https://docs.gitbook.com/published-documentation/customization/space-customization#layout-manage-navigation-options-for-your-content

**Reply by JaniceManwiller** [reply]
Thanks - When I looked at the header config I saw the **Link** text, but I didn't realize that it was showing the type of display. I wouldn't think to click that to toggle between link and button.
